lib/esp/resources/stat_region.rb

```diff
@@ -38,22 +37,22 @@ def self.for_stat(stat_id = nil, options = {}) # rubocop:disable Style/OptionHas
 
     # Find a StatRegion by id
     #
-    # ==== Parameter
+    # *call-seq* -> +super.find(id, options = {})+
     #
-    # +id+ | Required | The ID of the region stat to retrieve
+    # @overload find(id)
+    #   @param id [Integer, Numeric] Required ID of the region stat to retrieve.
+    #   @param options [Hash] Optional hash of options.
+    #     ===== Valid Options
     #
-    # +options+ | Optional | A hash of options
+    #     +include+ | The list of associated objects to return on the initial request.
     #
-    # ===== Valid Options
+    #     ===== valid Includable Associations
     #
-    # +include+ | The list of associated objects to return on the initial request.
-    #
-    # ===== valid Includable Associations
-    #
-    # See {API documentation}[http://api-docs.evident.io?ruby#stat-region-attributes] for valid arguments
-    #
-    # :call-seq:
-    #  find(id, options = {})
+    #     See {API documentation}[http://api-docs.evident.io?ruby#stat-region-attributes] for valid arguments
+    # @overload find(non_numeric, options = {})
```

Comment:
Not sure about documenting this overload, but if we do, it should be @overload find(options = {})